integration/client/export_test.go

```diff
@@ -60,7 +60,7 @@ func TestExport(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	// Seet to begining of file before passing it to assertOCITar()
+	// Seet to beginning of file before passing it to assertOCITar()
```

Comment:
"Seet" is also a typo

Reply:
@AkihiroSuda Thank you for your information.
I also fixed that.